src/Admin/ActivationRedirect.php

```diff
@@ -28,6 +29,19 @@ class ActivationRedirect implements Activateable, Service, Registerable, Options
 		'page' => 'wc-admin',
 		'path' => '/google/start',
 	];
+	/**
+	 * @var WPProxy
+	 */
+	protected $wp_proxy;
+
+	/**
+	 * Installer constructor.
+	 *
+	 * @param WPProxy $wp_proxy
+	 */
+	public function __construct( WPProxy $wp_proxy ) {
+		$this->wp_proxy = $wp_proxy;
+	}
```

Comment:
❓  Why this WPProxy naming?

I saw that in 

GlobalSiteTag.php, MerchantMetrics.php, WooCommerceBrands.php ...etc. (basically all the files using this WP class) are using  it as

`$this->wp = $wp;` instead of using WPProxy naming.

Reply:
👏🏻  Good spot. I named it that because it's clearer to me what the class does, but I'll swap it back for consistency with other classes.

Reply:
Updated in https://github.com/woocommerce/google-listings-and-ads/pull/1164/commits/7ae11d9d10d3582287bd2f68757629c8293c1c40